deploy/runtools/firesim_topology_elements.py

```diff
@@ -279,7 +279,8 @@ def copy_back_job_results_from_run(self, slotno):
                     rfsname = """/home/centos/sim_slot_{}/{}""".format(simserverindex, rfsname)
 
                 run("""sudo mount {blockfile} {mntpt}""".format(blockfile=rfsname, mntpt=mountpoint))
-                run("""sudo chmod -Rf 777 {}""".format(mountpoint))
+                run("""sudo chattr -i {}/etc/sysconfig/nfs""".format(mountpoint))
```

Comment:
There may be other files with the immutable bit.

```suggestion
                run("""sudo chattr -R -i {}""".format(mountpoint))
```